src/codegen/templates/constants.mako

```diff
@@ -0,0 +1,26 @@
+<%
+import utilities.enum_helpers as enum_helpers
+
+enums = enum_helpers.get_enums(data)
+%>\
+# Do not edit this file; it was automatically generated.
+
+from enum import Enum
+
+# Constants
+AUTO = -1
+CFG_DEFAULT = -1
+DEFAULT = -1
+READ_ALL_AVAILABLE = -1
+WAIT_INFINITELY = -1.0
+
+
+# Enums
+% for name, enum_metadata in enums.items():
+class ${name}(Enum):
+%   for value in enum_metadata['values']:
+    ${value['name']} = ${value['value']}${enum_helpers.get_enum_value_docstring(value, data)}
+%   endfor
```

Comment:
This should use python_name when available.

```suggestion
% for name, enum_metadata in enums.items():
class ${enum_metadata.get('python_name', name)}(Enum):
%   for value in enum_metadata['values']:
    ${value.get('python_name', value['name'])} = ${value['value']}${enum_helpers.get_enum_value_docstring(value, data)}
%   endfor
```

Reply:
Updated logic in enum_helpers.py to get `'python_name'` or `'name'`.